lib/identity_doc_auth/acuant/responses/get_results_response.rb

```diff
@@ -19,8 +19,12 @@ def initialize(http_response, config)
             extra: {
               result: result_code.name,
               billed: result_code.billed,
+              processed_alerts: processed_alerts,
+              alert_failure_count: processed_alerts[:failed].count,
```

Comment:
do we need to make this nil-safe?

```suggestion
              alert_failure_count: processed_alerts[:failed]&.count.to_i,
```

Reply:
I don't think so, but better safe than sorry.